projects/saturn/src/main/resources/application.yaml

```diff
@@ -35,6 +35,7 @@ jwt:
       principal-attribute: preferred_username
 
 application:
+  basePath: /api
```

Comment:
"/api" hardcoded value not replaced in all the places with this variable

Reply:
done, parametrized all controllers

relevance is to be discussed